src/setup_tests.ts

```diff
@@ -4,17 +4,12 @@
  * SPDX-License-Identifier: MIT
  */
 
-const util = require('util');
-
-// Required for the jest runtime
-TextDecoder = util.TextDecoder;
-
 // browser mocks
 const setLang = (lang: string) => {
   const localStorageMock = (function () {
     let store = {
       isTesting: true,
-      lang: lang,
+      lang: JSON.stringify(lang),
```

Comment:
This was a bug in the mocks that showed up in the test logs as they expected the language in local storage to be JSON stringified.

Reply:
I think the file has been broken for some time, but I think this was used by jest, which is removed now

Removed from package.json:
```json
    "setupFilesAfterEnv": [
      "<rootDir>/src/setup_tests.ts",
      "jest-expect-message"
    ],
```